switched.c

```diff
@@ -237,6 +261,34 @@ int main(int argc, char **argv) {
         case Instr_Break:
             cpu.state = Cpu_Break;
             break;
+        case Instr_Div:
+            tmp1 = pop(&cpu);
+            tmp2 = pop(&cpu);
+            BAIL_ON_ERROR();
+            if (tmp2 == 0) {
+                cpu.state = Cpu_Break;
+                break;
+            }
+            push(&cpu, tmp1 / tmp2);
+            break;
+        case Instr_Greater:
```

Comment:
This also could have been done as `Jump if greater` instruction - it would be more versatile. But let us keep `Greater` for now.